modules/nw-multus-configure-dualstack-ip-address.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-additional-network.adoc
+
+:_content-type: PROCEDURE 
+
+[id="nw-multus-configure-dualstack-ip-address_{context}"]
+= Creating a configuration for assignment of dual-stack IP addresses dynamically
+
+Dual-stack IP address assignment can be configured with the `ipRanges` parameter for:
+* IPv4 addresses
+* IPv6 addresses
+* multiple IP address assignment 
+
+.Procedure
+
+. Set type to whereabouts.
+. Use `ipRanges` to allocate IP addresses.
+
+[source,yaml]
+----
+cniVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  additionalNetworks:
+  - name: whereabouts-shim
+    namespace: default
+    rawCNIConfig: |-
+      {
+       "name": "whereabouts-dual-stack",
+       "cniVersion": "0.3.1,
+       "type": "bridge",
+       "ipam": {
+         "type": "whereabouts",
+         "ipRanges": [{
+           "range": "192.168.10.1/24"
+         }, {
+           "range": "2001:db8::/64"
+         }]
+       }
+      }
+
+
+----
+
+. Attach network to a pod. For more information, see "Adding a pod to an additional network".
+
+. Verify that all IP addresses are assigned.
+
+Run the following two commands to ensure the IP adresses are assigned as meta-data.
+
+[source,yaml]
+----
+oc exec -it mypod -- ip a
+----
+
+[source,yaml]
+----
+oc describe pod mypod
```

Comment:
I would either get the specific network-status by specifying jsonpath - i.e. `oc get -ojsonpath ...`  or put some grep to directly show the interesting data.

Reply:
If you want a different code set, I will need that directly from you.

Reply:
I suggested it in https://github.com/openshift/openshift-docs/pull/66173#discussion_r1360362398

Let me know if that makes sense to you. I think this is more straight-forward to the user, but it might raise some questions (having to escape some characters, etc).

Also, let's get @dougbtv 's opinion.